libr/debug/p/debug_gdb.c

```diff
@@ -575,7 +575,6 @@ static const char *r_debug_gdb_reg_profile(RDebug *dbg) {
 			"seg	gs	.32	60	0\n"
 			);
 		}
-		break;
```

Comment:
Wtf

Reply:
Every if-else condition in that case has a return in it so the break is unnecessary.